tests/Resources/fs.addontest.js

```diff
@@ -0,0 +1,1493 @@
+					should(err).be.ok; // aka, there is an error
+					// TODO Verify the error.code value is EACCESS!
+					finished();
+				} catch (e) {
+					finished(e);
+				}
+			});
+		});
+
+		it('uses mode F_OK by default', finished => {
+			fs.access(thisFilePath, err => {
+				finished(err);
+			});
+		});
+
+		it('checks that this file is readable properly', finished => {
+			fs.access(thisFilePath, fs.constants.R_OK, err => {
+				finished(err);
+			});
+		});
+
+		it('checks that this file is NOT writable properly', finished => {
+			fs.access(thisFilePath, fs.constants.W_OK, err => {
+				try {
+					should(err).be.ok; // aka, there is an error
+					// TODO Verify the error.code value is EACCESS!
+					finished();
+				} catch (e) {
+					finished(e);
+				}
+			});
+		});
+
+		it('checks that this file is NOT executable properly', finished => {
+			fs.access(thisFilePath, fs.constants.X_OK, err => {
+				try {
+					should(err).be.ok; // aka, there is an error
+					// TODO Verify the error.code value is EACCESS!
+					finished();
+				} catch (e) {
+					finished(e);
+				}
+			});
+		});
+	});
+
+	describe('#accessSync()', () => {
+		it('is a function', () => {
+			should(fs.accessSync).be.a.Function;
+		});
+
+		it('checks that this file exists properly', () => {
+			fs.accessSync(thisFilePath, fs.constants.F_OK);
+		});
+
+		it('throws when trying to access file that doesn\'t exist', () => {
+			should.throws(() => {
+				fs.accessSync('/madeup');
+			}, Error);
+			// TODO: Verify format of error!
+		});
+
+		it('uses mode F_OK by default', () => {
+			fs.accessSync(thisFilePath);
+		});
+
+		it('checks that this file is readable properly', () => {
+			fs.accessSync(thisFilePath, fs.constants.R_OK);
+		});
+
+		it.allBroken('checks that this file is NOT writable properly', () => {
+			// FIXME: This isn't throwing an error like we expect!
+			// How can we test this? resourcesDirectory is not supposed to be writable on device, but can be on simulator.
+			// (and this may be platform-specific behavior)
+			should.throws(() => {
+				fs.accessSync(thisFilePath, fs.constants.W_OK);
+			}, Error);
+			// TODO Verify the error.code value is EACCESS!
+		});
+
+		it('checks that this file is NOT executable properly', () => {
+			should.throws(() => {
+				fs.accessSync(thisFilePath, fs.constants.X_OK);
+			}, Error);
+			// TODO Verify the error.code value is EACCESS!
+		});
+	});
+
+	// TODO: #appendFile()
+
+	describe('#appendFileSync()', () => {
+		it('is a function', () => {
+			should(fs.appendFileSync).be.a.Function;
+		});
+	});
+
+	// TODO: #close()
+
+	describe('#closeSync()', () => {
+		it('is a function', () => {
+			should(fs.closeSync).be.a.Function;
+		});
+
+		it('returns undefined', () => {
+			const fd = fs.openSync(thisFilePath);
+			const result = fs.closeSync(fd);
+			should(result).be.undefined();
+		});
+	});
+
+	describe('#copyFile()', () => {
+		it('is a function', () => {
+			should(fs.copyFile).be.a.Function;
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/f...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/fs.addontest.js;
  Line: 176;
-->

- :warning: tests/Resources/fs.addontest.js line 176 – The 'fs.copyFile' is not supported until Node.js 8.5.0. The configured version range is '>=8'. (node/no-unsupported-features/node-builtins)